app/templates/views/privacy.html

```diff
@@ -63,12 +63,11 @@ <h2 class="heading-medium">How long we keep your data</h2>
 
     <h2 class="heading-medium">Where your data is processed and stored</h2>
 
-    <p class="govuk-body">We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s
+    <p class="govuk-body">We design, build and run our systems to make sure that your data is as safe as possible at every stage, both while it’s
```

Comment:
Might need to revert this one – I’m waiting for confirmation.

```suggestion
    <p class="govuk-body">We design, build and run our systems to make sure that your data is as safe as possible at any stage, both while it’s
```